steps/property-inspection/antisocial-behaviour.tsx

```diff
@@ -73,9 +75,9 @@ const step: ProcessStepDefinition<ProcessDatabaseSchema, "property"> = {
           key: "paragraph-1",
           Component: Paragraph,
           props: {
-            children: `Antisocial behaviour is defined as "behaviour by a 
-            person which causes, or is likely to cause, harassment, alarm or 
-            distress to one or more persons not of the same household as the 
+            children: `Antisocial behaviour is defined as "behaviour by a
+            person which causes, or is likely to cause, harassment, alarm or
+            distress to one or more persons not of the same household as the
```

Comment:
It would be worth checking that the changes here don't remove the spaces between the words at the end of each line and the beginning of the following line. It may be easier to use single quotes instead, like so:

```
children: 'Antisocial behaviour is defined as "behaviour by a person which causes, or is likely to cause, harassment, alarm or distress to one or more persons not of the same household as the person".'
```